analysis/clinvar/clinvar_variation_analysis.ipynb

```diff
@@ -1159,6 +1159,33 @@
    "cell_type": "code",
    "execution_count": 23,
    "metadata": {},
+   "outputs": [],
+   "source": [
+    "variation_type_count_supported_df.to_csv(\n",
+    "    \"variation_analysis_output/variation_type_count_supported_df.csv\"\n",
+    ")"
+   ]
+  },
+  {
```

Comment:
```
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### <a id='toc1_4_2_'></a>[Supported and Not Normalized Variants](#toc0_)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "supported_not_normalized_df = df.copy()"
   ]
  },
   ```
    These lines should be removed. Everything below is for *Supported and Normalized Variants* but this section looks like it was copied from down below on lines 1731-1746 giving this first section the appearance it is for *Not normalized* when it is for *normalized*

Reply:
oops. found this issue. from the past

Reply:
@jarbesfeld  The set of sections following the section noted above seem to be a bit odd and hard to understand. It looks like the headings do not align with the tables. Please double check and make sure I'm not hallucinating.

Reply:
@larrybabb @korikuzma I had some merge conflicts that I must have not resolved correctly. I just pulled in the changes from main and re-ran, so this should look normal now